provider/galaxy.go

```diff
@@ -57,16 +57,21 @@ func (g AnsibleGalaxy) VersionsForRole(ctx context.Context, r requirements.Role)
 		keywords = r.Name
 	}
 
+	// set the Ansible Galaxy search parameters
+	params := url.Values{}
+
 	// namespace to be used to filter the Ansible Galaxy results
 	var namespace string
 	split := strings.Split(keywords, ".")
 	if len(split) > 0 {
 		namespace = split[0]
+		params.Add("namespaces", namespace)
```

Comment:
Is this block https://github.com/atosatto/ansible-requirements-lint/pull/3/commits/38012e07e6fc0b496582b5c319cb4e6975653072#diff-0b318929670f1f6ac9487ff469a8afe3R120-R132 still required now that we pass the namespace as parameter of the Ansible Galaxy request or is sufficient to take the first returned results?

Reply:
I think we could remove the else block.